RecoHI/HiCentralityAlgos/plugins/HiHFFilterProducer.cc

```diff
@@ -0,0 +1,100 @@
+
+#include <memory>
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/CaloTowers/interface/CaloTower.h"
+#include "DataFormats/CaloTowers/interface/CaloTowerCollection.h"
+#include "DataFormats/HeavyIonEvent/interface/HFFilterInfo.h"
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescriptionFiller.h"
+
+#include "DataFormats/Common/interface/Wrapper.h"
+
+#include <cmath>
+
+class HiHFFilterProducer : public edm::stream::EDProducer<> {
+public:
+  explicit HiHFFilterProducer(const edm::ParameterSet&);
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+private:
+  void produce(edm::Event&, const edm::EventSetup&) override;
+  edm::EDGetTokenT<CaloTowerCollection> srcTowers_;
+};
+
+HiHFFilterProducer::HiHFFilterProducer(const edm::ParameterSet& iConfig)
+    : srcTowers_(consumes<CaloTowerCollection>(iConfig.getParameter<edm::InputTag>("srcTowers"))) {
+  produces<reco::HFFilterInfo>("hiHFfilters");
+}
+
+void HiHFFilterProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
+  using namespace std;
+  using namespace edm;
+
+  auto const& towers = iEvent.get(srcTowers_);
+
+  unsigned short int nTowersTh2HFplus = 0;
+  unsigned short int nTowersTh2HFminus = 0;
+  unsigned short int nTowersTh3HFplus = 0;
+  unsigned short int nTowersTh3HFminus = 0;
+  unsigned short int nTowersTh4HFplus = 0;
+  unsigned short int nTowersTh4HFminus = 0;
+  unsigned short int nTowersTh5HFplus = 0;
+  unsigned short int nTowersTh5HFminus = 0;
+
+  auto HFFilterResults = std::make_unique<reco::HFFilterInfo>();
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.

Reply:
implemented in commit 61df60f895814facc634bc83e8b3d7bcb97788d2